kadi/commands/command_sets.py

```diff
@@ -194,6 +189,8 @@ def cmd_set_end_scs(*args, date=None):
 
 def cmd_set_command_not_run(*args, date=None):
     (cmd,) = cmd_set_command(*args, date=date)
+    # Save original type which gets used later in CommandTable.remove_not_run_cmds().
+    cmd["params"]["__type__"] = cmd["type"]
```

Comment:
This seems fine, though the name `__type__` wasn't intuitive for me.    But I suppose anything else would get pretty long __not_run_orig_type__ etc.